Tools/Cli-LoRa-Device-Provisioning/Keygen.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Cli_LoRa_Device_Provisioning
+{
+    using System;
+
+    public static class Keygen
+    {
+        public static string Generate(int keyLength)
+        {
+            var randomKey = string.Empty;
+            var rnd = new Random();
+
+            for (int i = 0; i < keyLength; i++)
+            {
+                var newKey = rnd.Next(0, 255);
```

Comment:
since upper bound is exclusive, you would never get 255 but only 254.

Reply:
Cough... Didn't know that. Thx!